SETUP_GUIDELINES.md

```diff
@@ -0,0 +1,34 @@
+## Steps for Developers 
+1. Download the latest stable release of [Android Studio](https://developer.android.com/studio/index.html).
+2. Fork the [Project Repository](https://github.com/openfoodfacts/openfoodfacts-androidapp) by clicking "Fork".
+3. Clone the repository <br>`git clone https://github.com/YOUR_USERNAME/openfoodfacts-androidapp.git`.
+4. Change directory `cd openfoodfacts-androidapp`.
+5. Run the project in Android Studio
+6. Take a look at [README.md](https://github.com/openfoodfacts/openfoodfacts-androidapp/blob/master/README.md)
```

Comment:
@PrajwalM2212  i didn't meant to put the Readme.md link here.  I meant to put this `Setup-guidelines.md` link inside Readme  So users can navigate.   you can put it in  #Installation. 
You can use this:-   `If you're a new contributor to the open-source, We recommend you to read our [_SETUP-GUIDELINES_](Link_Here)`

Reply:
Ok , I get it . Thanks for the help @sachincool